src/test/run-make-fulldeps/coverage-reports/expected_show_coverage_counters.async.txt

```diff
@@ -33,8 +33,6 @@ Counter in file 0 7:9 -> 7:10, #2
 Counter in file 0 9:9 -> 9:10, (#1 - #2)
 Counter in file 0 11:1 -> 11:2, (#2 + (#1 - #2))
 Counter in file 0 21:1 -> 21:23, #1
-Counter in file 0 67:5 -> 67:23, #1
```

Comment:
These changes I'm not sure are OK, but I'm not sure why they are changing either. 

I'm not sure if this will break cross-platform tests.

Might require a closer look.

Reply:
@Swatinem - I think you are safe if the `expected_show_coverage_counters.*` files change by sort order.

It looks like the Makefile does not fail the test if the `_counters` files don't match. There are a couple of reasons why those files aren't stable enough, mostly due to sort order.